app/helpers/feature_flag.rb

```diff
@@ -1,7 +1,15 @@
 module FeatureFlag
   extend self # rubocop:disable Style/ModuleFunction
 
+  def strict_enabled?(feature_name, *args)
+    Flipper.enabled?(feature_name, *args)
+  end
+
+  def exist?(feature_name)
+    Flipper.exist?(feature_name)
+  end
+
   def enabled?(feature_name, *args)
-    Flipper[feature_name].enabled?(*args)
+    !exist?(feature_name) || strict_enabled?(feature_name, *args)
```

Comment:
Maybe we can do something like instead:

```ruby
def enabled?(feature_name, *args)
  Flipper.enabled?(feature_name, *args)
end

def exist?(feature_name)
  Flipper.exist?(feature_name)
end

def active?
  # what the current enabled? is trying to do
end
```

If we do this we can also use delegates to wrap the `Flipper` class methods:

```ruby
module FeatureFlag
  class << self
    delegate :enabled?, :exists?, to: Flipper

    def active?
      ...
    end
  end
end 
```

BTW, not sure about returning true for missing features, that seems a bit counterintuitive to me. The only way I'd expect something to not show up is when it's (1) gated AND (2) enabled.

Reply:
How could I have forgotten about `delegate` 🤦 Thanks, this reads much cleaner.

I'm pretty firm that the name of the class should remain FeatureFlag: a Feature is a much more generic term. I like how specific FeatureFlag is.

Reply:
> I'm pretty firm that the name of the class should remain FeatureFlag

Sure, let's stick to the name `FeatureFlag`. I sometimes just throw ideas out during reviews, unless I use the "request changes" feature (almost never) it's always meant as a suggestion that can be ignored.